src/Core/BoundedContexts/Headquarters/WB.Core.BoundedContexts.Headquarters/Storage/ContentTypeHelper.cs

```diff
@@ -0,0 +1,58 @@
+using System.IO;
+
+namespace WB.Core.BoundedContexts.Headquarters.Storage;
+
+public static class ContentTypeHelper
```

Comment:
Please consider usage of FileExtensionContentTypeProvider from Microsoft.AspNetCore.StaticFiles or MimeMapping.GetMimeMapping(fileName) from System.Web.
Do we need two methods?